plugins/animation/index.js

```diff
@@ -0,0 +1,15 @@
+const plugin = require('tailwindcss/plugin')
+
+module.exports = plugin(({ addUtilities, theme }) => {
+  const pluginConfig = theme('animation', {})
+
+  const utilities = Object.entries(pluginConfig).map(([name, value]) => ({
+    [`.animation-${name}`]: {
```

Comment:
I kind of think we should always be escaping selectors, just in case someone provides a special character in the name. I could see something like this happening: `.spin-1/2` and `.spin-1/4`.

Reply:
Great call